docs/prometheus-integration.md

```diff
@@ -156,6 +156,11 @@ detected by Flow Exporter deny connections tracking. This metric gets updated
 when a flow is rejected/dropped by network policy.
 - **antrea_agent_egress_networkpolicy_rule_count:** Number of egress
 NetworkPolicy rules on local Node which are managed by the Antrea Agent.
+- **antrea_agent_flow_exporter_to_flow_aggregator_count:** Number of
```

Comment:
I think this metric should be maintained in the flow aggregator code and have Prometheus exporter in there. We cannot maintain this at the flow exporter as it can only individual agents.
May be the name `flow_aggregator_flow_exports_count` is more appropriate.

I understand that you are still working on it. Thought I should point this.

Reply:
Currently for Prometheus metrics, we only have support for Antrea Controller and Agent. Maybe it is not necessary to register a new client and scraping configuration only for exporting number of exporters connected. Since we now have a number of connections per collector information at Flow Aggregator, we can log it periodically along with other information. What do you think?